pkg/seccomp/default_linux.go

```diff
@@ -573,6 +674,19 @@ func DefaultProfile() *Seccomp {
 				Caps: []string{"CAP_SYS_NICE"},
 			},
 		},
+		{
+			Names: []string{
+				"get_mempolicy",
+				"mbind",
+				"set_mempolicy",
+			},
```

Comment:
Not sure why those are separate from `move_pages` and `migrate_pages` above?

Reply:
these fails with EPERM only when the container is not granted CAP_SYS_NICE.  Otherwise seccomp won't block them

Reply:
Hmm. `move_pages` and `migrate_pages` seem to have similar constraints. I also believe that these system calls do not require `CAP_SYS_NICE` in all cases. See the discussion of the `MPOL_MF_MOVE_ALL` flag.

Reply:
do you suggest enabling them in any case?

Reply:
As far as I know, they are similar to `madvise` and not really privileged, so I think they can be enabled.

Reply:
thanks.  Added a patch to always allow them